frontend/src/app.vue

```diff
@@ -95,7 +97,7 @@ const app = {
       users: [],
       userUpdated: false,
 
-      isLoadingOverlayEnabled: false,
+      isLoadingOverlayEnabled: null,
```

Comment:
Why null works but not false?

Reply:
(deprecation ATTR_FALSE_VALUE) Attribute "aria-busy" with v-bind value `false` will render aria-busy="false" instead of removing it in Vue 3. To remove the attribute, use `null` or `undefined` instead.

Reply:
`aria-busy` seems to be defined in the `vue-loading-overlay` package, try to check the github repo for this issue.

Reply:
My guess for why this issue occurred is as we are still using the old css for `vue-loading-overlay` as seen [here](https://github.com/reposense/RepoSense/blob/ed131f6294bf83fac436a6265bc4d53b022739f7/frontend/src/app.vue#L322)

Reply:
`false` is indeed a valid value. I think I will revert it back first and if it is an issue after removing the compatible build then I will fix it again.